fe2-android/app/src/test/ui/robolectric/com/github/dedis/popstellar/ui/detail/EventListAdapterTest.java

```diff
@@ -123,21 +123,18 @@ public void updateAreDisplayed() {
 
     // Default values
     events.onNext(Sets.newSet(ROLL_CALL, ROLL_CALL2));
-
+    System.out.println(adapter.getItemCount());
     assertEquals(EventListAdapter.TYPE_HEADER, adapter.getItemViewType(0));
-    assertEquals(EventListAdapter.TYPE_HEADER, adapter.getItemViewType(1));
+    assertEquals(EventListAdapter.TYPE_EVENT, adapter.getItemViewType(1));
     assertEquals(EventListAdapter.TYPE_EVENT, adapter.getItemViewType(2));
-    assertEquals(EventListAdapter.TYPE_EVENT, adapter.getItemViewType(3));
-    assertEquals(EventListAdapter.TYPE_HEADER, adapter.getItemViewType(4));
 
     // Open rollcall 1 should move it from
-    events.onNext(Sets.newSet(RollCall.openRollCall(ROLL_CALL), ROLL_CALL2));
+    events.onNext(Sets.newSet(RollCall.closeRollCall(ROLL_CALL), ROLL_CALL2));
```

Comment:
either this was changed by mistake, or the comment was not updated :-)

Reply:
The comment is updated now :)